src/sass/_slider.scss

```diff
@@ -0,0 +1,30 @@
+.carousel__img {
+  min-width: 100%;
+  object-fit: cover;
+  height: 300px;
+}
+
+.carousel-item {
+  height: 300px;
+
+  &::after {
+    display: block;
```

Comment:
`@extend .d-block, .position-relative, .mt-n5, .h-100, .w100;` would be a nice replacement (not sure about the amount of negative margin needed there)